src/components/studies/LiveIcon.tsx

```diff
@@ -9,27 +9,24 @@ const useStyles = makeStyles(theme => ({
     backgroundColor: '#51A3A3',
     outline: 'none',
     display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'center',
     color: 'white',
     fontWeight: 'bold',
+    flexDirection: 'row',
+    justifyContent: 'space-evenly',
+    alignItems: 'center',
+    padding: `0px ${theme.spacing(0.5)}px`,
```

Comment:
You don't need to do '`0px ${theme.spacing(0.5)}px`'   -   padding: theme.spacing(0, .5 ),is preferable. Here and throughout